### DISCUSSION TITLE
minimum PHP version

### DISCUSSION THREAD

**Discussion by pildit** [pestphp/pest#503] [General]
Hi guys, 

in the Pest docs it says that Pest requires PHP 7.3+ 

but in the Pest composer.json I noticed this line :

`"php": "^8.0",`

After all, which is the minimum version ? 

Thanks

**Comment by owenvoke** [comment]
PHP 8.0 is the minimum version for Pest 2.x (which is on the `master` branch). Check the `1.x` branch for v1 (which is PHP 7.3+). The docs haven't been updated for Pest 2.x yet.

**Reply by pildit** [reply]
Thanks
